docs/reference/msbuild-targets.md

```diff
@@ -431,6 +433,17 @@ msbuild -t:build -restore
 
 The same logic applies to other targets similar to `build`.
 
+### Restoring PackageReference and packages.config with MSBuild
+
+With MSBuild 16.5+, packages.config are also supported for `msbuild -t:restore`.
+
+```cli
+msbuild -t:restore -p:RestorePackagesConfig=true
+```
+
+> [!NOTE]
+> `packages.config` restore is only available with `MSBuild 16.5+`, and not with `dotnet.exe`
+
```

Comment:
What do you think of adding an example of using `Directory.Build.props` to make this a more permanent setting for a particular project? It's rare that folks would want to only do this once, and having to write it every time seems like a lot.

Reply:
oh, and I'm not sure that a `Directory.Build.props` file would do it, because in general these customers will be building the solution, not projects individually, and I don't think that `Directory.Build.props` will get imported for a solution restore. I recently learned that `Directory.Solution.props` is a thing, and maybe that would work, I don't know. But I still don't understand why customers would do this multiple times when the packages.config experience is so tied to Visual Studio.

Reply:
oh I didn't @zivkan's comment...it'd have to be in a directory with/above solution the solution. 

Directory.Solution.Props would work too. 

I'll leave it as is for now :) 

Maybe we can consider that in a new issue.